src/components/text/TextBit.jsx

```diff
@@ -44,7 +45,7 @@ const TextBit = ({
   ...props}: TextBitPropsType) => {
   const Type = type;
   const textClass = classNames('sg-text-bit', {
-    [`sg-text-bit--${size || ''}`]: size,
+    [`sg-text-bit--${size || ''}`]: size !== TEXT_BIT_SIZE.NORMAL,
```

Comment:
Why not use:

```
const TextBit = ({ . . . , size = TEXT_BIT_SIZE.NORMAL }) => { . . . }
```

So you don't have to have the conditional `${size || ''}`. By default if no `size` prop is passed, it will be normal.